scapy/contrib/gtp.py

```diff
@@ -193,13 +195,13 @@ def answers(self, other):
                 self.version == other.version and
                 self.payload.answers(other.payload))
 
-    @classmethod
-    def dispatch_hook(cls, _pkt=None, *args, **kargs):
-        if _pkt and len(_pkt) >= 1:
-            if (struct.unpack("B", _pkt[0])[0] >> 5) & 0x7 == 2:
-                import gtp_v2
-                return gtp_v2.GTPHeader
-        return GTPHeader
+    @classmethod

```

Comment:
Files that have wrong line endings: gtp.py, ppi.py, ppi_cace.py, ppi_geotag.py, rsvp.py
Editing those files often creates messy eol like this one :(